data/json/effects_on_condition/npc_eocs/hallucination_eocs.json

```
@@ -209,19 +214,11 @@
     "condition": { "math": [ "u_monsters_nearby()", ">", "1" ] },
     "effect": [
       {
-        "if": { "math": [ "u_i", "<", "rng(50, 100)" ] },
-        "then": [
-          {
-            "u_spawn_monster": "",
-            "hallucination_count": 1,
-            "target_range": 50,
-            "max_radius": 50,
-            "lifespan": [ "2 hours", "9 hours" ]
-          },
-          { "math": [ "u_i", "++" ] },
-          { "run_eocs": "hallucination_swarm" }
-        ],
-        "else": { "math": [ "u_i", "=", "0" ] }
+        "u_spawn_monster": "",
+        "hallucination_count": [ 50, 100 ],
```

```suggestion
        "hallucination_count": [ 50, 100 ],
```
This does result in a higher likelihood of 75+ hallucinations than the way it was working before if that's not desired

---

No, it's fine